Content.Server/Power/EntitySystems/SubstationSystem.cs

```diff
@@ -0,0 +1,384 @@
+using System.Diagnostics.CodeAnalysis;
+using Robust.Server.GameObjects;
+using Robust.Shared.Timing;
+using Robust.Shared.Containers;
+using Content.Server.Power.Components;
+using Content.Server.Construction;
+using Content.Server.Atmos.Components;
+using Content.Server.Atmos;
+using Content.Shared.Power.Substation;
+using Content.Shared.Access.Systems;
+using Content.Shared.Rejuvenate;
+using Content.Shared.Examine;
+using Content.Shared.Atmos;
+
+namespace Content.Server.Power.EntitySystems;
+
+public sealed class SubstationSystem : EntitySystem 
+{
+    
+    [Dependency] private readonly PointLightSystem _lightSystem = default!;
+    [Dependency] private readonly SharedPointLightSystem _sharedLightSystem = default!;
+    [Dependency] private readonly AppearanceSystem _appearance = default!;
+
+
+    private bool _substationDecayEnabled = true;
```

Comment:
```suggestion
{
    [Dependency] private readonly PointLightSystem _lightSystem = default!;
    [Dependency] private readonly SharedPointLightSystem _sharedLightSystem = default!;
    [Dependency] private readonly AppearanceSystem _appearance = default!;

    private bool _substationDecayEnabled = true;
```

Reply:
Also you should either prefix all or none of these dependencies with `System`, so it's consistent across the file.